bridge/github/import.go

```diff
@@ -23,61 +23,46 @@ const (
 // githubImporter implement the Importer interface
 type githubImporter struct {
 	conf core.Configuration
+
+	// iterator
+	iterator *iterator
+
+	// number of imported issues
+	importedIssues int
+
+	// number of imported identities
+	importedIdentities int
 }
 
 func (gi *githubImporter) Init(conf core.Configuration) error {
 	gi.conf = conf
 	return nil
 }
 
+func (gi *githubImporter) Reset() {
+	gi.importedIssues = 0
+	gi.importedIdentities = 0
+}
+
 // ImportAll .
 func (gi *githubImporter) ImportAll(repo *cache.RepoCache, since time.Time) error {
-	iterator := NewIterator(gi.conf[keyUser], gi.conf[keyProject], gi.conf[keyToken], since)
+	gi.iterator = NewIterator(gi.conf[keyUser], gi.conf[keyProject], gi.conf[keyToken], since)
```

Comment:
Was forced to do this since the `ImportAll`, `ensureIssue` and `ensurePerson`  functions need to have access to `importedIssues`/`importedIdentities` variables